### DISCUSSION TITLE
Transaction not in mempool

### DISCUSSION THREAD

**Discussion by cliffpeskin** [lightningnetwork/lnd#7226] [Troubleshooting]
I am running an LND node on a Ubuntu Server with a Neutrino backend and have a utxo for 1,593,279 sats with 0 confirmations that will not appear in the mempool.

Transaction ID is **a705f0411200305482ce7610819c3c44a5cf75a04d3f3451ce9319a433add9e9**

If anyone could share some suggestions on how to resolve that would be great.  

![image](https://user-images.githubusercontent.com/20306099/205361533-cca2236d-863b-472b-9cb2-80f8ce63a21b.png)

**Comment by guggero** [comment]
Did you try a restart? Lnd will re-broadcast the transaction and with the now empty mempool it should be picked up.

**Comment by cliffpeskin** [comment]
Hi @guggero, yes I've tried restarting LND several times but unfortunately that hasn't helped.

**Reply by guggero** [reply]
Do you see the transaction listed in the output when you run `lncli listchaintxns`? Do you see any error related to publishing the transaction in the lnd log when you start it up?

**Reply by cliffpeskin** [reply]
Yes, I see the transaction in the output for `lncli listchaintxns`.

```
 lncli listchaintxns
{
    "transactions": [
        {
            "tx_hash": "a705f0411200305482ce7610819c3c44a5cf75a04d3f3451ce9319a433add9e9",
            "amount": "1593279",
            "num_confirmations": 0,
            "block_hash": "",
            "block_height": 0,
            "time_stamp": "1670005082",
            "total_fees": "0",
            "dest_addresses": [
                "bc1p4ceq55su2n4u33lwekpveyu8xv0j29722qyx8sqmkemayrxuexqszyhtll"
            ],
            "output_details": [
                {
                    "output_type": "SCRIPT_TYPE_PUBKEY_HASH",
                    "address": "bc1p4ceq55su2n4u33lwekpveyu8xv0j29722qyx8sqmkemayrxuexqszyhtll",
                    "pk_script": "5120ae320a521c54ebc8c7eecd82cc9387331f2517ca500863c01bb677d20cdcc981",
                    "output_index": "0",
                    "amount": "1593279",
                    "is_our_address": true
                }
            ],
            "raw_tx_hex": "02000000000102a6512ad3697ec0f5ee631e896d7283aef19080c8f2bc80da52b8ee0b735b6f3001000000000000000099150dc85f0526428efba4d2922f6c5174056c1e7e33d80f32711cd97f9e71f300000000000000000001bf4f180000000000225120ae320a521c54ebc8c7eecd82cc9387331f2517ca500863c01bb677d20cdcc98102483045022100df2f528737e9e22c362a6357713b796b1bd371ddf395cc2de73245e49462631c022057201c7229836e8e3312ab321850979985bc8a6961e20eb15551f6f6d8d3ad3f01282103fe1c6c6b1702e6f578e2e1ad5829748a134ea91effed7cbe8b25bb33c6ea67d0ac736460b26801401962f3d37c69a5c515547439ad01f59f0a7a72aa30470770a905d9351a29d5c19361650b94bfe911ecc1f0bc633f113bf5d3e74997f206626f104215989006f1a8a60b00",
            "label": "0:sweep",
            "previous_outpoints": [
                {
                    "outpoint": "306f5b730beeb852da80bcf2c88090f1ae83726d891e63eef5c07e69d32a51a6:1",
                    "is_our_output": false
                },
                {
                    "outpoint": "f3719e7fd91c71320fd8337e1e6c0574516c2f92d2a4fb8e4226055fc80d1599:0",
                    "is_our_output": false
                }
            ]
        },
```

When searching transaction ID in lnd logs, I don't see any error messages but I do these the following:

`2022-11-17 05:58:25.813 [INF] SWPR: Creating sweep transaction a705f0411200305482ce7610819c3c44a5cf75a04d3f3451ce9319a433add9e9 for 2 inputs (306f5b730beeb852da80bcf2c88090f1ae83726d891e63eef5c07e69d32a51a6:1 (CommitmentAnchor), f3719e7fd91c71320fd8337e1e6c0574516c2f92d2a4fb8e4226055fc80d1599:0 (TaprootPubKeySpend)) using 253 sat/kw, tx_weight=725, tx_fee=0.00000183 BTC, parents_count=0, parents_fee=0 BTC, parents_weight=0`

`2022-11-17 05:58:25.815 [INF] LNWL: Inserting unconfirmed transaction a705f0411200305482ce7610819c3c44a5cf75a04d3f3451ce9319a433add9e9`

`2022-12-02 11:37:11.812 [INF] LNWL: Inserting unconfirmed transaction a705f0411200305482ce7610819c3c44a5cf75a04d3f3451ce9319a433add9e9`

**Reply by guggero** [reply]
Okay, I see the input for this transaction was already swept, so the TX cannot be accepted. What version of lnd are you on? I think this should eventually resolve itself. If not, you can try `--reset-wallet-transactions` (which takes a loooooong time with Neutrino).

**Comment by cliffpeskin** [comment]
I am using lnd version 0.15.4-beta. I will try `--reset-wallet-transactions`. Thanks @guggero !
